client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -28,4 +31,22 @@ export class EmployeeListComponent implements OnInit, OnDestroy {
   ngOnInit() {}
 
   ngOnDestroy() {}
+
+  updateEmployeeStatus(employee: Employee) {
+    this.service
+      .updateEmployeeStatus(employee)
+      .pipe(
+        catchError((err) => {
+          console.log(`Could not update employee ${employee.id}`);
+          return throwError(err);
+        }),
+      )
+      .subscribe({
+        next: (detail) => {
+          this.toastr.show('Employee sucessfully updated!');
```

Comment:
Normally I'd do this kind of stuff in the service and try to keep the component as thin as possible.

Think of the component as a Controller + View in MVC or Controller + JSON in an API.

Reply:
fixed according to the discussion we had this morning, moved all of this code to the service